docs/api/products.rst

```diff
@@ -1,33 +1,168 @@
+
++----------------+----------------------------------------------------------------------------+
+| Field          | Description                                                                |
++================+============================================================================+
+| id             | Id of product                                                              |
++----------------+----------------------------------------------------------------------------+
+| code           | Unique product identifier                                                  |
++----------------+----------------------------------------------------------------------------+
+| average_rating | Average from ratings given by customer                                     |
++----------------+----------------------------------------------------------------------------+
+| channels       | Collection of channels to which product was assigned                       |
++----------------+----------------------------------------------------------------------------+
+| translations   | Collection of translations (each contains slug and name in given language) |
++----------------+----------------------------------------------------------------------------+
+
+If you request for a more detailed data, you will receive an object with following fields:
+
++----------------+----------------------------------------------------------------------------+
+| Field          | Description                                                                |
++================+============================================================================+
+| id             | Id of product                                                              |
++----------------+----------------------------------------------------------------------------+
+| code           | Unique product identifier                                                  |
++----------------+----------------------------------------------------------------------------+
+| average_rating | Average from ratings given by customer                                     |
++----------------+----------------------------------------------------------------------------+
+| channels       | Collection of channels to which product was assigned                       |
++----------------+----------------------------------------------------------------------------+
+| translations   | Collection of translations (each contains slug and name in given language) |
++----------------+----------------------------------------------------------------------------+
+| attributes     | Collection of attributes connected with product (for example material)     |
++----------------+----------------------------------------------------------------------------+
+| variants       | Collection of variants connected with product                              |
```

Comment:
We are missing reviews, available on, available until, options, product taxons, main taxon and reviews and images. 

IMHO options, main taxon and product images should be part of defualt serialization